src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
     };
 
     const renderObjectOverview = () => {
-        const startTimeInMilliseconds = Number(currentSchemaData?.CreateStep);
-        const createdAt = startTimeInMilliseconds
-            ? formatDateTime(startTimeInMilliseconds)
-            : 'unknown';
-        const createdAtLabel = 'Created At';
-
-        // verbose mapping to guarantee a correct render for new path types
-        // TS will error when a new type is added but not mapped here
-        const pathTypeToComponent: Record<EPathType, (() => React.ReactNode) | undefined> = {
```

Comment:
What do you think about not to delete this mapping? I think it's rather useful when add a new type.
In the past it was a headache to remember all places where we need to add info about new type.

Reply:
For old variant we are creating a new component for any type separately. But I've rewrote the logic to have common info and adding addition info for certain type. 

May be it is possible to rewrite with `pathTypeToInfo` function with Record<>, but I don't understand the meaning of this